app/assets/javascripts/Components/markdown_preview.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export default class MarkdownPreview extends React.Component {
+  componentDidMount = () => {
+    const target_id = "annotation_preview";
+    MathJax.Hub.Queue(["Typeset", MathJax.Hub, target_id]);
+  };
+
+  componentWillUnmount = () => {
+    setTimeout(this.props.updateAnnotationCompletion(), 300);
+  };
+
+  render() {
+    return (
+      <div
+        id="annotation-preview"
+        style={{
```

Comment:
Remove this style. The preview `div` had its own class with some CSS attached to it (which you should keep).